jsk_fetch_robot/jsk_fetch_startup/launch/visualize_trash.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
```

Comment:
do we need this launch?

Reply:
Sorry, I mistakenly added it.
I removed it in 5235bdd.

Reply:
If I should remove this file from the commit log, I will remove it with force-pushing.

Reply:
you can do force pushing, if you want.

Reply:
Thank you. I removed this from commit log.